index.html

```diff
@@ -889,6 +889,17 @@
 <aside class="issue">
   Write this.
 </aside>
+
+To the extent possible, [=people=] should retain a certain number of rights over their data, and
+these rights should be facilitated by the [=parties=]. While data rights are a self-governing
+mechanism that is not sufficient to support [=privacy=], they do complement self-determination and
+help improve accountability. Such rights should include:
```

Comment:
Carve-outs for counterabuse are common regarding these rights, and should be recognized here. Specifically, users who have a history of criminal activity or other acts of abuse may be limited in their right to demand such history be erased. Restrictions to access signal data used in counterabuse enforcement is also generally necessary. Perhaps this is implied by "To the extent possible". Any objections to making these allowances explicit?